File: PY/exoalgo.py
from pickle import TRUE
import random
print("EXO 1")
a = 5
print(a)

print("EXO 2")
print
a = 7
b = 17
result = a + b
print(result)
result = a - b
print(result)
result = a * b
print(result)

print("EXO 3")
a = "hello mate\n"
print(a)

print("EXO 4")
a = "coucou"
b = " petite perruche"
print(a + b)

print("Exo 5")
a = 20 
b = 30 
z = b
b = a
a = z
print(a, b)

print("Exo 6")
a = 20 
b = 30 
# Code à ajouter ci-dessous 
if(a > b):
    print("A est supérieur à B")


elif(b > a):
    print("B est supérieur à A")

else:
    print("Les chiffres sont égaux")

print("Exo 7")
a = 20
b = 20
print("La valeur A = " + str(a))
print("La valeur B = " + str(b))
def maFonction():
    if(a > b):
        print("A est supérieur à B")
    elif(b > a):
        print("B est supérieur à A")
    else:
        print("Les chiffres sont égaux")

maFonction()

print("Exo 8")
print(random.randint( 0, 10))

print("Exo 9")
a = 0
while(a < 10):
    print(str(random.randint(1,20))) 
    a += 1

print("Exo 10")
a = 0
while (a < 90):
    a = random.randint(1,100)
    print(a)
if(a >= 90):
    print("C'est finis")

print("EXO 11")
a = input("Quel est votre mot ? ")
print("Vous avez choisit le mot :")
print(a)

print('EXO 12')
a = int(input('Entrez un nombre ici :'))
print("Vous avez choisis le nombre : " + str(a))
if(a % 2 == 0):
    print("Votre nombre est un nombre pair")
elif(a % 2 >= 0):
    print('Votre nombre est impair')

print("EXO 13")
v = input('Entrez un nombre ici :')

if(v.isnumeric() == False):
    print('ERREUR: Vous devez entrer un nombre')
elif(v % 2 == 0):
    print("Votre nombre est un nombre pair")
    print("Vous avez choisis le nombre : " + str(v))
elif(v % 2 >= 0):
    print('Votre nombre est impair')
    print("Vous avez choisis le nombre : " + str(v))

print('EXO 14')
m = input("Entrez une année : ")
if(m %4  == 0):
    print("C'est une année Bisextile")
else:
    print("C'est une année Sextile")
print('EXO 15')
print("Entrez une date : ")
t = input()

File: JS/exoalgo.js
/* EXO 1 */
var a = 5;
console.log('EXO1 La variable est ' + a);

/* EXO 2 */
var a = 10;
var b = 12;
console.log('EXO2: a + b = ' + (a + b));
console.log('a - b = ' + (a - b));
console.log('a x b = ' + (a * b));
/* EXO 3 */
var z = "EXO03: Bonjour, ceci est une variable";
console.log(z);
/* EXO 4 */
var a = "EXO04: Coucou ";
var b = "Petite perruche";
console.log(a + b);
/* EXO 5 */
var a = 30;
var b = 20;
console.log("EXO5: a = 30");
console.log("b = 20")
var c = a;
var a = b;
var b = c;
console.log("A = " + a + '  B = ' + b);
/* EXO 6 */
var a = 30;
var b = 20;
if(a > b) {
    console.log('EXO 6 : A est plus grand que B');
}
else if (b > a) {
    console.log('EXO 6 : B est plus grand que A');
}
else {
    console.log("EXO 6 : Les nombre sont égaux.")
}
/* EXO 7 */
var a = 30;
var b = 20;
function maVariable(x, y) {
    if (x > y) {
        console.log("EXO07 : " + x + " Est plus grand que " + y);
    }
    else if (y > x) {
        console.log("EXO07 : " + y + " Est plus grand que " + x);
    }
    else {
        console.log("EXO07 : Les nombres sont égaux");
    }
    
}
maVariable(a, b);
/* EXO 8 */
function getRandomInt(max) {
    return Math.floor(Math.random() * max);
}
console.log(getRandomInt(10));

/* EXO 9 */
console.log('EXO 9');

for( i = 0; i < 10; i++) {
    console.log(getRandomInt(100));
}
/* EXO 10 */
var a = 0;
console.log('EXO 10');
while (a < 90) {
    a = getRandomInt(100);
    console.log(a);
}
/* EXO  11 */
/* "Afficher une variable entrée manuellement par l'utilisateur */
/* Code à ajouter ci-dessous */
console.log("EXO 11 ");
var a = prompt('Quel est votre variable ? ');
console.log("La variable que vous avez choisit est : " + a);

/* EXO n°12 */
/*Demander à l'utilisateur d'entrer un nombre entier et afficher si ce chiffre est pair ou impair*/

/* Code à ajouter ci-dessous */
console.log("EXO 12 ");
var b = prompt('Quel est votre variable ? ');
console.log("La variable que vous avez choisit est : " + b);
if( b %2 == 0) {
    console.log("La variable que vous avez choisit est pair");
}
else {
    console.log("La variable que vous avez choisit est impair");
}


/*  EXO n°13 */
console.log("EXO 13");
/*Même exercice que n°12, mais rajouter un message d'erreur si l'utilisateur n'entre pas un nombre entier */
/* Code à ajouter ci-dessous */
var d = prompt('Quel est votre variable ? ');
if(isNaN(d)) {
    console.log("Loupé, vous devez inscrire un nombre ");
}
else{
    console.log("votre nombre à bien été pris en compte");
}
console.log("La variable que vous avez choisit est : " + d);
if( b %2 == 0) {
    console.log("La variable que vous avez choisit est pair");
}
else {
    console.log("La variable que vous avez choisit est impair");
}

/* EXO n°14 */
/* Demander à l'utilisateur d'entrer une année, et afficher si cette année est sextile ou bissextile*/
/* Code à ajouter ci-dessous */
console.log("EXO 14");
var j = prompt("Veuillez écrire une année");
if (j %4 == 0) {
    console.log("Vous êtes dans une année Bisextile");
}
else if (j %4 > 0) {
    console.log("Vous êtes dans une année Sextile");
}

/* EXO n°15*/
/*Demander à l'utilisateur d'entrer une date, et afficher si cette date est valide ou non */
/* Code à ajouter ci-dessous */